django_fido/migrations/0003_add_authenticator.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-24 09:09
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
I am wondering if there is a data migration possible or not.

Reply:
It may be possible, but we decided not to do it.

Reply:
Ok.